public/themes/theme_4.json

```diff
@@ -1,119 +1,220 @@
+          "settings": {
+            "values": {
+              "sectionTitle": "About the Institution",
+              "showInNavigation": true,
+              "bgColor": "#f07057",
+              "color": "#ffffff"
+            },
+            "form": [
+              {
+                "type": "settings",
+                "items": [
+                  {
+                    "type": "checkbox",
+                    "label": "Show in navigation",
+                    "value": "showInNavigation",
+                    "validation": null
+                  }
                 ]
-            }
+              }
+            ]
+          }
+        },
+        {
+          "sectionHeader": "Image",
+          "component": "layer-image",
+          "settings": {
+            "values": {
+              "height": 500,
+              "url": "http://localhost:4200/img/building.jpg",
```

Comment:
Is this the photo of German parliament? Can you note somewhere where you got it, if its unsplash you don't have to.